dvc/command/plots.py

```diff
@@ -123,7 +124,12 @@ def add_parser(subparsers, parent_parser):
         "--template",
         nargs="?",
         default=None,
-        help="File to be injected with data.",
+        help=(
+            "Special JSON or HTML schema to inject with the data. "
```

Comment:
Oops, maybe also important to add the word "file" in there.
```suggestion
            "Special JSON or HTML schema file to inject with the data. "
```

Reply:
https://github.com/iterative/dvc/pull/3854